components/Wallet/index.jsx

```diff
@@ -20,33 +20,19 @@ import {
 } from '../../utils/ledger/reportLedgerConfigError'
 import MsgConfirmer from '../../lib/confirm-message'
 import useWallet from '../../WalletProvider/useWallet'
-import { MESSAGE_HISTORY, SEND } from './views'
 import reportError from '../../utils/reportError'
 
 export default () => {
   const wallet = useWallet()
   const router = useRouter()
-  const childView = router.pathname.includes('home') ? MESSAGE_HISTORY : SEND
   const { ledger, connectLedger } = useWalletProvider()
   const [uncaughtError, setUncaughtError] = useState('')
   const [showLedgerError, setShowLedgerError] = useState(false)
   const [ledgerBusy, setLedgerBusy] = useState(false)
 
-  const setChildView = useCallback(
-    view => {
-      const params = new URLSearchParams(router.query)
-      if (view === MESSAGE_HISTORY) {
-        router.push(`/home?${params.toString()}`)
-      } else if (view === SEND) {
-        router.push(`/send?${params.toString()}`)
-      }
-    },
-    [router]
-  )
-
-  const onAccountSwitch = () => {
-    const params = new URLSearchParams(router.query)
-    router.push(`/home/accounts?${params.toString()}`)
+  const resetWallet = () => {
+    // a full page reload will reset the wallet
+    window.location.reload()
```

Comment:
Not a critical change, but moving this to a function helps explain why we reload -- to reset the wallet.